GLTF/src/GLTFTargetNames.cpp

```diff
@@ -0,0 +1,26 @@
+#include "GLTFTargetNames.h"
+
+#include "rapidjson/stringbuffer.h"
+#include "rapidjson/writer.h"
+
+std::string GLTF::MorphTargetNames::typeName()
+{
+	return "MorphTargetNames";
+}
+
+void GLTF::MorphTargetNames::writeJSON(void* writer, GLTF::Options* options)
+{
+	GLTF::Object::writeJSON(writer, options);
+	rapidjson::Writer<rapidjson::StringBuffer>* jsonWriter = (rapidjson::Writer<rapidjson::StringBuffer>*)writer;
+	jsonWriter->StartArray();
+	for (std::string morph : morphs)
```

Comment:
I would use a reference here for `morph`, to avoid string objects being created, but that is really not a big issues here, given the small number of morph targets